app/utils/roadmap-sorting.ts

```diff
@@ -0,0 +1,53 @@
+import seedrandom from 'seedrandom';
+
+export function getRoughWeekSeed(date: number): number {
+  const now = date;
+  const weekMs = 7 * 24 * 60 * 60 * 1000; // milliseconds per week
+
+  return now - (now % weekMs);
+}
+
+export function getSortPositionForRoadmapPage(
+  developmentStatus: string,
+  votesCount: number,
+  id: string,
+  date: number,
+  currentUserId?: string | null,
+): string {
+  const statusPriority =
```

Comment:
```suggestion
  const statusSortKey =
```

(We can make this a string right away, no need to call `.toString()`, and adding `sortKey` to the name makes it clear that it's used in `sortKeys` below and avoids the additional "priority" word)